ship.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  shouldPrepare: ({ releaseType, commitNumbersPerType }) => {
+    const { fix = 0 } = commitNumbersPerType;
+    if (releaseType === 'patch' && fix === 0) {
+      return false;
+    }
+    return true;
+  },
+  testCommandBeforeRelease: () =>
+    'yarn run lint && yarn run test && yarn run test:e2e',
```

Comment:
Is it necessary? In the CI we only release after tests no?

Reply:
Ah you're right, I didn't think about that. I'll remove it, thanks for noticing!